docs/csharp/language-reference/keywords/using.md

```diff
@@ -3,7 +3,6 @@ description: "using keyword - C# Reference"
 title: "using keyword - C# Reference"
 ms.date: 04/05/2019
 f1_keywords: 
-  - "using_CSharpKeyword"
   - "using"
```

Comment:
If this keyword won't be generated by Visual Studio F1, I'd like to remove this article. It has no content, and it's only purpose is to provide a single click to send readers where they want to get to.

We'd also need to check for links and remove links to this page.

@davidwengier do you know if "using" is generated from Visual Studio?

Reply:
Right now, Roslyn appends `_CSharpKeyword` for any language construct it is trying to access help for, so unless there is an old version of Visual Studio that might be triggering this (that we care about) so as far as the `f1_keywords` goes I don't expect this to be ever used.

Reply:
What about users browsing the documentation outside of F1-help?  If they were to go to the [keywords](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/) page, what should they see?  I think leaving in the `using` page for that purpose makes sense.  There's no "using-statement" _keyword_, they would expect a "using" keyword page.

Reply:
Ping @BillWagner in case you didn't see this comment.

I'm happy from the Roslyn side of this, and the PR on that side is ready to go when this is in, but this question is for the docs point of view.

Reply:
Sorry, missed this comment.  Thanks for the ping.

This is good to go, and I'll :shipit: now.